pkgs/development/python-modules/xdot/default.nix

```diff
@@ -11,9 +11,18 @@ buildPythonPackage rec {
   };
 
   disabled = !isPy3k;
-
   nativeBuildInputs = [ wrapGAppsHook ];
-  propagatedBuildInputs = [ gobject-introspection pygobject3 graphviz gtk3 ];
+  propagatedBuildInputs = [ gobject-introspection pygobject3 graphviz gtk3 numpy ];
+  checkInputs = [ xvfb_run ];
+
+  checkPhase = ''
+    xvfb-run -s '-screen 0 800x600x24' ${python3.interpreter} nix_run_setup test
+  '';
+
+  # https://github.com/NixOS/nixpkgs/pull/107872#issuecomment-752175866
+  doCheck = stdenv.isLinux;
+  # The importsCheck seems to run the checkPhase, but still fails to init gtk?
+  # pythonImportsCheck = [ "xdot" ];
```

Comment:
The comment is wrong. the importsCheck does something similar to: ``python -c "import xdot"``.

Reply:
Thanks, I've [removed those two lines.](https://github.com/NixOS/nixpkgs/compare/2d091edabce03058078812a9cfb171b94614d122..6d355e44f8274b8c0142d50bfdaec168db3ebd24)